### PR TITLE
[KOB-3907] Merged all plugins

### PR DESCRIPTION
**Pull request by tiennguyenhoan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,17 @@
-# xlr-kobiton-plugin
-XebiaLabs Release plugin for integrating with Kobiton
+# Kobiton plugin for XL-Release
+
+## Preriquisites
```
Comment (duylam) [inline README.md]: Setup for development

File: README.md [hunk 2]
```diff
@@ -1,2 +1,17 @@
-# xlr-kobiton-plugin
-XebiaLabs Release plugin for integrating with Kobiton
+# Kobiton plugin for XL-Release
```
Comment (duylam) [inline README.md]: The Kobiton plugin for XebiaLabs XL Release product

File: README.md [hunk 3]
```diff
@@ -1,2 +1,17 @@
...
+
+## Preriquisites
+All the commands below are running on MacOSX
```
Comment (duylam) [inline README.md]: - Assume the environment is Mac OS
- Run below commands

File: README.md [hunk 4]
```diff
@@ -1,2 +1,17 @@
...
+- Install Jython: `brew install jython`
```
Comment (duylam) [inline README.md]: Miss: install brew

File: README.md [hunk 5]
```diff
@@ -1,2 +1,17 @@
...
+- Install [IntelliJ IDEA Community Edition](https://www.jetbrains.com/idea/download/#section=mac)
+- Install [JDK 1.7](http://www.oracle.com/technetwork/java/javase/downloads/java-archive-downloads-javase7-521261.html)
+
+## Building
```
Comment (duylam) [inline README.md]: `Build and deploy`

File: README.md [hunk 6]
```diff
@@ -1,2 +1,17 @@
...
+- Run command `gradle clean build` at the root directory
+- Copy the `jar` from `build/libs` into `plugins/__local__` directory in XL-Release server
+
+## Types
```
Comment (duylam) [inline README.md]: Remove this

File: build.gradle [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+// In this section you declare where to find the dependencies of your project
+repositories {
+    // Use jcenter for resolving your dependencies.
+    // You can declare any Maven/Ivy/file repository here.
+    jcenter()
+    mavenLocal()
+    mavenCentral()
+}
```
Comment (duylam) [inline build.gradle:38]: Can we remove `gradle/wrapper/gradle-wrapper.jar` and note for download in setup step ?

File: gradlew.bat [hunk 8]
```diff
@@ -0,0 +1,84 @@
...
+:fail
+rem Set variable GRADLE_EXIT_CONSOLE if you need the _script_ return code instead of
+rem the _cmd.exe /c_ return code!
+if  not "" == "%GRADLE_EXIT_CONSOLE%" exit 1
+exit /b 1
+
+:mainEnd
+if "%OS%"=="Windows_NT" endlocal
+
+:omega
```
Comment (duylam) [inline gradlew.bat]: Can we remove all files in `/lib/` ?

**Review by duylam — COMMENTED** [review]
@NguyenHoanTien comment, em

File: build.gradle (hunk 7 above)
Comment (tiennguyenhoan) [reply build.gradle:38]: This is come from the template of XL, I'm not sure it cause any impact when we delete this

**Pull request closed without merging**